Modules/Core/Common/test/itkSpatialOrientationTest.cxx

```diff
@@ -0,0 +1,113 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkSpatialOrientation.h"
+#include <set>
+
+
+int
+itkSpatialOrientationTest(int, char *[])
+{
+
+  // Test streaming enumeration for SpatialOrientationEnums::CoordinateTerms elements
+  const std::set<itk::SpatialOrientationEnums::CoordinateTerms> allCoordinateTerms{
```

Comment:
Is there a reason why those enum values stored in an `std::set`, before printing them? It appears to me that these values might as well be passed directly to the `for` loop, like the following:

```
for (const auto enumValue : { itk::SpatialOrientationEnums::CoordinateTerms::ITK_COORDINATE_UNKNOWN,
                              itk::SpatialOrientationEnums::CoordinateTerms::ITK_COORDINATE_Right,
                              // ...
                              itk::SpatialOrientationEnums::CoordinateTerms::ITK_COORDINATE_Superior })

```

Just my two cents, Niels

Reply:
I know this has been requested and changed in https://github.com/InsightSoftwareConsortium/ITK/pull/3155#r798926614, but ALL the rest of the serializations have been done using an `std::set`. I am not against the way you propose, but that should be done in a separate PR, across the whole code, and the ITK SW Guide be changed accordingly: https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/pull/155.

Reply:
> I know this has been requested and changed in [#3155 (comment)](https://github.com/InsightSoftwareConsortium/ITK/pull/3155#r798926614),

So fortunately, I'm consistent with my comments  😺 Always trying to avoid redundant and unnecessary code.